core/services/database/TableManager.php

```diff
@@ -184,6 +184,12 @@ public function dropIndex($table_name, $index_name)
      */
     public function createTable($table_name, $create_sql, $engine = 'MyISAM')
     {
+        $engine = apply_filters(
+            'FHEE__EventEspresso__core__services__database__TableManager__createTable__engine',
```

Comment:
doesn't our filter naming schema only use single underscores to replace directory separators?

I think this should be: `FHEE__EventEspresso__core_services_database_TableManager__createTable__engine` no?